ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py

```diff
@@ -1,3 +1,184 @@
 # global
+from hypothesis import strategies as st
 
 # local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+# relu
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.relu",
```

Comment:
The path is incorrect in all of the tests. Use path `paddle.nn.functional.<func_name>` here for example `paddle.nn.functional.relu`. Kindly make this change in all of the tests in this file